dev/com.ibm.ws.security/src/com/ibm/ws/security/krb5/Krb5Common.java

```diff
@@ -36,16 +35,10 @@
     static public Oid KRB5_MECH_OID;
 
     // Is IBM JDK 1.8
-    static public boolean isIBMJdk18 = (JavaInfo.vendor() == Vendor.IBM && JavaInfo.majorVersion() == 8);
-
-    // Is Oracle JDK
-    static public boolean isOracleJdk = (JavaInfo.vendor() == Vendor.ORACLE);
-
-    // Is IBM, Oracle and Open JDK 11 or higher
-    static private boolean isJdk11OrUp = JavaInfo.majorVersion() >= 11;
+    static public boolean isIBMJdk18 = JavaInfo.isAvailable("com.ibm.security.auth.module.Krb5LoginModule");
 
     // SPNEGO support IBM JDK 8 and lower and JDK 11 and higher
-    static public boolean isOtherSupportJDKs = isOracleJdk || isJdk11OrUp;
+    static public boolean isOtherSupportJDKs = JavaInfo.isAvailable("com.sun.security.auth.module.Krb5LoginModule");
```

Comment:
Variable names should change.

Reply:
Sure, separate PR though

Reply:
https://github.com/OpenLiberty/open-liberty/issues/18055